packages/agent-toolkit/src/core/tool.ts

```diff
@@ -22,6 +22,7 @@ export interface Tool<Input extends ZodRawShape | undefined, Output extends Reco
   name: string;
   type: ToolType;
   annotations: ToolAnnotations;
+  enabledByDefault?: boolean;
```

Comment:
worth mentioning what is the default behavior. i.e. are all tools enabled by default or disabled by default?